### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by johnnymatthews**
This PR just tidies spelling and grammar.

### CODE REVIEW

File: cmd/relay-gossip/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ TODO: will be fixed soon in https://github.com/drand/drand/pull/501
 
 #### Failover
 
-The `-url` and `-failover-grace` flags can be used when relaying over gRPC to provide 1 or more alternative HTTP API endpoints that may be able to provide randomness in the event of a failure of the gRPC connection, and the amount of time to leave after a round is due before they are contacted. e.g.
+The `-url` and `-failover-grace` flags can be used when relaying over gRPC. These flags provide one or more alternative HTTP API endpoints that may be able to provide randomness in the event of a failure of the gRPC connection. The amount of time to leave after a round is due before they are contacted. e.g.
```
Comment (johnnymatthews) [inline cmd/relay-gossip/README.md:69]: @alanshaw does this sentence make sense in the context of this doc?

Comment (alanshaw) [reply cmd/relay-gossip/README.md:69]: Ok how about:

```suggestion
The `-url` and `-failover-grace` flags can be used in combination when relaying over gRPC. The `-url` flag provides the URL(s) of alternative HTTP API endpoints that may be able to provide randomness in the event of a failure of the gRPC connection. The `-failover-grace` flag enables failover to these endpoints and sets the amount of time to leave after a round is due before they are contacted. e.g.
```

File: cmd/relay-gossip/README.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ func main() {
 
 #### With Known Chain Hash
 
-You do not need to know the full group info to use the pubsub client, if you know the chain hash and a HTTP endpoint then you can request the chain info from the HTTP endpoint, verifying it with the known chain hash:
+You do not need to know the full group info to use the pubsub client if you know the chain-hash and an HTTP endpoint then you can request the chain info from the HTTP endpoint, verifying it with the known chain hash:
```
Comment (alanshaw) [inline cmd/relay-gossip/README.md:141]: Why is chain hash hyphenated here?

File: cmd/relay-gossip/README.md (hunk 1 above)
Comment (johnnymatthews) [reply cmd/relay-gossip/README.md:69]: That'll do.

File: cmd/relay-gossip/README.md (hunk 2 above)
Comment (johnnymatthews) [reply cmd/relay-gossip/README.md:141]: If it's different to a [hash chain](https://en.wikipedia.org/wiki/Hash_chain) then I think it should be hyphenated. If _hash chain_ and _chain hash_ are the same thing, then I suggest we use _chain hash_ across the docs.

Comment (alanshaw) [reply cmd/relay-gossip/README.md:141]: It is not hash chain 😄.

It's a hash of the information relating to the chain of randomness that's being generated by drand i.e. the public key, period and genesis time. I do not know if that necessitates a hyphen?

Comment (johnnymatthews) [reply cmd/relay-gossip/README.md:141]: Ok cool. So yeah you're right, doesn't need the hyphen. I think I threw another one in there somewhere. I'll find it and change it back.

Comment (johnnymatthews) [reply cmd/relay-gossip/README.md]: I've left the hyphen in `chain hash` on lines [21](https://github.com/drand/drand/pull/564/files#diff-bc07a701f06bee4379b40c8d8b9e47b6R21) and [31](https://github.com/drand/drand/pull/564/files#diff-bc07a701f06bee4379b40c8d8b9e47b6R31), it looks like they're addresses or something?

**Pull request merged**